about.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="icon" href="./Images/CODING_OUTDOORS.png" type="image/png" sizes="16x16">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+        type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+        type="text/css" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./css/main.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>CODE&OUTDOORS</title>
+</head>
+
+<body>
+    <!-- Header -->
+    <nav id="header">
+        <div class="header container">
+            <div class="nav-bar d-flex">
+                <div class="logo">
+                    <a href="index.html">
+                        <h1><span>C</span>ODE <SPan>O</SPan>UTDOORS</h1>
+                    </a>
+                </div>
+                <div class="nav-list">
+                    <div class="hamburger">
+                        <a class="menu-icon1"><i class="fa fa-bars"></i></a>
+                        <a class="menu-icon2 d-none"><i class="fa fa-times color-white"></i></a>
+                    </div>
+                    <ul class="d-flex flex-dir-col">
+                        <li class="nav-item">
+                            <a href="index.html" data-after="Home">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="about.html" data-after="About">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./ExplorePage/index.html" data-after="Tieckts">Tickets</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <!-- End of Header -->
+    <!--     hero section -->
+    <section id="hero">
+        <div class="hero container d-flex flex-dir-col">
+            <div class="hero-title">
+                <h2>Coding & Travelling<span></span></h2>
+                <h2>Meeting <span></span></h2>
```

Comment:
- [ ] You need to have at least one highest heading text `h1` tag and this one looks the right one for this page. Please do the same for the other page too.

Reply:
Done thanks for your review 👍